sql/static/README.md

```diff
@@ -0,0 +1,4 @@
+This directory contains definitions for derived tables in the `static` dataset.
+
+In particular, we have two tables based on a one-time export of Firefox Accounts
+extracted from Amplitude and imported to BigQuery.
```

Comment:
should we do likewise for the telemetry dataset, so it's clear which dataset a given sql file is intended for?

Reply:
I like it.